configuring/using-builds-in-a-network-restricted-environment.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="using-builds-in-a-network-restricted-environment"]
+include::_attributes/common-attributes.adoc[]
```

Comment:
attributes include file should be after the title.
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata